pkg/ccl/backupccl/datadriven_test.go

```diff
@@ -12,6 +12,7 @@ import (
 	"context"
 	gosql "database/sql"
 	"fmt"
+	"github.com/cockroachdb/cockroach/pkg/ccl/backupccl/backuputils"
```

Comment:
still looks misplaced, should probably be in the stanza of imports below

Reply:
Done.